src/slurm_plugin/instance_manager.py

```diff
@@ -86,16 +88,23 @@ def add_instances_for_nodes(
         for queue, compute_resources in instances_to_launch.items():
             for compute_resource, slurm_node_list in compute_resources.items():
                 logger.info("Launching instances for slurm nodes %s", print_with_count(slurm_node_list))
+
+                # each compute resource can be configured to use create_fleet or run_instances
+                fleet_manager = FleetManagerFactory.get_manager(
+                    self._cluster_name,
+                    self._region,
+                    self._boto3_config,
+                    self._cluster_config_file,
+                    queue,
+                    compute_resource,
+                    all_or_nothing_batch,
+                )
+                launch_overrides = self._launch_overrides.get(queue, {}).get(compute_resource, {})
+
                 for batch_nodes in grouper(slurm_node_list, launch_batch_size):
```

Comment:
It cannot be removed because we need to use a batch size when adding the items in route53. 
Yes in future we could have different size for the different apis.